File: UNIT 4/Python/matriks2d.py
def main():
    matriks = [[None for i in range(4)] for j in range(3)]
    
    print("Input elemen array: \n")
    for baris in range(0, 3):
        for kolom in range(0, 4):
            print("matriks[{}][{}]".format(baris + 1, kolom + 1))
            matriks[baris][kolom] = int(input())
        print()
        
    print("Isi array: \n")
    for baris in range(3):
        for kolom in range(4):
            print(matriks[baris][kolom], end=" ")
        print()

main()

File: UNIT 4/C/matriks2d.c
#include "stdio.h"


int main() {
    /*Kamus Lokal*/
    int baris, kolom, matriks[3][4];
    
    /*Algoritma*/
    printf("Program Output Matriks\n\n");
    printf("Input elemen Array: \n");
    
    for (baris = 0; baris < 3; baris++) {
        for (kolom = 0; kolom < 4; kolom++) {
            printf("matriks[%i][%i]>>> ", baris + 1, kolom + 1);
            scanf("%i", &matriks[baris][kolom]);
        }
        printf("\n");
    }
    
    printf("matriks :\n");
    for (baris = 0; baris < 3; baris++) {
        for (kolom = 0; kolom < 4; kolom++) {
            printf("%i ", matriks[baris][kolom]);
        }
        printf("\n\n");
    }
    
   
    return 0;
}